Google.Cloud.Spanner.NHibernate.Tests/SpannerDriverTest.cs

```diff
@@ -1394,31 +1394,31 @@ private StatementResult CreateTableWithAllColumnTypesResultSet(TableWithAllColum
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Float64}, "colfloat2_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Numeric}, "colnumeric3_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Bool}, "colbool4_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.String}, "colstring5_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.String}, "colstringmax6_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Bytes}, "colbytes7_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Bytes}, "colbytesmax8_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Date}, "coldate9_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Timestamp}, "coltimestamp10_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Json}, "coljson11_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Timestamp}, "colcommitts12_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Int64}}, "colint13_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Float64}}, "colfloat14_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Numeric}}, "colnumericarray15_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Bool}}, "colboolarray16_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.String}}, "colstringarray17_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.String}}, "colstringmaxarray18_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Bytes}}, "colbytesarray19_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Bytes}}, "colbytesmaxarray20_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Date}}, "coldatearray21_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Timestamp}}, "coltimestamparray22_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Array, ArrayElementType = new V1.Type {Code = V1.TypeCode.Json}}, "coljsonarray23_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Bytes}, "colcomputed24_2_0_"),
-                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Bytes}, "asc25_2_0_")
+                Tuple.Create(new V1.Type{ Code = V1.TypeCode.Int64}, "colint1_4_0_"),
```

Comment:
nit: You can avoid the need to update all these references if you add the new entities to the mapping __after__ the `TableWithAllColumnTypes` mapping.

The reason is that NHibernate will assign a unique sequence number to each entity that is added to the mapping, and include this sequence number in all queries that it generates. Before this change, `TableWithAllColumnTypes` had sequence number 2, because it was the third entity that was added to the mapping (the sequence numbers start at zero). So if you change the mapping so that the new entities are added as the last entities, then you won't need this change. (See also my comment above)